docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+database:
+  image: "sameersbn/mysql:latest"
+  environment:
+    DB_USER: staytus
+    DB_PASS: staytus
```

Comment:
Mmm, it shouldn't be directly exposed, but good security generally revolves around ensuring the whole tower of cards doesn't come tumbling down if any one thing fails. E.g. https://www.owasp.org/index.php/Defense_in_depth. It'd be good to be distributing software that's following security best practices, since people probably are going to put this straight into production. At the same time though, you're right that it's probably fine, and it's certainly not an actual vulnerability.

For now, if you can come up with a nice easy (i.e. no user interaction) way to generate a random password, that would be amazing. If nothing springs to mind though I think we should ignore this for now and create a new issue to think about doing it better later. It's not a big problem, more of a nice to have.

Maybe we could add a security note to the readme; if you're putting this into production, you should change the password below to minimize security risks? That'll probably do.